scripts/get-fonts.sh

```diff
@@ -131,3 +133,10 @@ mv "${FONTDIR}/static/NotoEmoji-Bold.ttf" "${FONTDIR}"
 curl --fail -A "get-fonts.sh/osm-carto" -o "${TMPDIR}/hanazono.zip" -L 'https://osdn.net/frs/redir.php?f=hanazono-font%2F68253%2Fhanazono-20170904.zip'
 
 unzip -oqq "${TMPDIR}/hanazono.zip" HanaMinA.ttf HanaMinB.ttf -d "${FONTDIR}"
+
+curl --fail -A "get-fonts.sh/osm-carto" -o "${TMPDIR}/unifont.tar.gz" -L 'https://unifoundry.com/pub/unifont/unifont-14.0.04/unifont-14.0.04.tar.gz'
+
+tar -xvf "${TMPDIR}/unifont.tar.gz" -C "${FONTDIR}/static/" unifont-14.0.04/font/precompiled/unifont-14.0.04.otf unifont-14.0.04/font/precompiled/unifont_upper-14.0.04.otf
```

Comment:
Use `--strip-components` to remove the directory from the extracted filename. This will avoid creating another temporary directory.

Reply:
Fixed.